redhat/README.md

```diff
@@ -0,0 +1,105 @@
+# Ceph-CSI Stream
+
+Ceph-CSI Stream is the Red Hat downstream project that contains the pre-release
+state of Ceph-CSI as used in the OpenShift Data Foundation product.
+
+## Git Repository
+
+### Branches
+
+This GitHub repository contains branches for different product versions.
+
+## Backports
+
+All changes in this repository are *backports* from the [upstream
+project][upstream-ceph-csi]. There should be no functional changes (only
+process/CI/building/..) in this repository compared to the upstream project.
+Fixes for any of the release branches should first land in the master branch
```

Comment:
devel branch?

Reply:
ah, yes, missed a few in the search/replace